src/components/Books.module.css

```diff
@@ -0,0 +1,119 @@
+.progCircle {
+  width: 5.625rem;
+  max-height: 5.625rem;
+}
+
+.smContainer {
+  position: relative;
+  bottom: 100px;
+  left: 600px;
+  display: flex;
+  flex-direction: row;
+}
+
+@import url("https://fonts.googleapis.com/css2?family=Montserrat:wght@300&display=swap");
+
+.container {
+  width: 75rem;
+  height: 10.625rem;
+  margin: 2.313rem 6.25rem 0;
+  padding: 2rem 9.188rem 1.625rem 1.688rem;
+  border-radius: 4px;
+  border: solid 1px #e8e8e8;
+  background-color: #fff;
+  list-style: none;
+  position: relative;
+  bottom: 20px;
+}
+
+.category {
+  width: 200px;
+  height: 1.125rem;
+  margin: 0 2.438rem 0 0;
+  opacity: 0.9;
+  font-family: "Montserrat", sans-serif;
+  font-size: 1rem;
+  font-weight: 600;
+  text-align: start;
+  color: hsl(0, 0%, 53%);
+}
+
+@import url("https://fonts.googleapis.com/css2?family=Roboto+Slab&display=swap");
```

Comment:
Please add all the @import rules at the top of the document, you can find why and more information on this [link](https://www.w3schools.com/cssref/pr_import_rule.asp).